vector/src/main/scala/geotrellis/vector/io/json/JsonFeatureCollection.scala

```diff
@@ -99,11 +99,11 @@ object JsonFeatureCollection{
     fc
   }
 
-  def apply(geometries: Traversable[Geometry]) = {
+/*  def apply(geometries: Traversable[Geometry]) = {
     val fc = new JsonFeatureCollection()
     fc ++= geometries.toList
     fc
-  }
+  }*/
 
```

Comment:
I think we discussed that lists/seq of pure Geometry types cannot be applied to a FeatureCollection, and therefore, this would need to be removed, right?